src/WeedSeeker.Web.Test.Functional/WeedSeeker.Web.Test.Functional.csproj

```diff
@@ -30,10 +30,26 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="nunit.core, Version=2.6.4.14350, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">

```

Comment:
OK... I get it now. That's the package that makes VS2013 run nUnit 3.0 tests inside its own UI, just like VS 2015 does by default.

I just accepted your pull request. 

Congratulations. You made it to the master. :smile: